src/ontology/pato-edit.obo

```diff
@@ -20588,6 +20588,34 @@ def: "Existing or functioning as though broken into separate parts; disorganized
 is_a: PATO:0001444 ! broken
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: PATO:0040048
+name: absent physical object
```

Comment:
Sounds a bit too much like it refers to a physical entity rather than a quality.  Maybe 'absence of physical object' ?

Reply:
I renamed this